Index: workdir/compile
===================================================================
--- workdir.orig/compile
+++ workdir/compile
@@ -9,13 +9,13 @@ PURPLE='\033[0;35m'
 NC='\033[0m' # No Color
 # sample : I ${RED}love${NC} WIMS
 
-if [ "$(id -ru)" = 0 ]; then
-  cat <<@
-You are trying to compile WIMS as 'root'. You should not do that !
-Please compile as 'wims'.
-@
-  exit 1
-fi
+#if [ "$(id -ru)" = 0 ]; then
+#  cat <<@
+#You are trying to compile WIMS as 'root'. You should not do that !
+#Please compile as 'wims'.
+#@
+#  exit 1
+#fi
 
 option=$1
 
